### PR TITLE
gh-161: bump coverage for galaxies and shapes

### PR DESCRIPTION
**Pull request by Saransh-cpp**
The coverage went down when branch coverage was turned on. This PR brings it back to 100% for these two files.

Refs: #161

### CODE REVIEW

**Comment by Saransh-cpp** [comment]
coverage above 60 🚀

File: tests/test_galaxies.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ def test_redshifts(mocker):  # type: ignore[no-untyped-def]
     assert z.shape == (10,)
 
 
-def test_redshifts_from_nz():  # type: ignore[no-untyped-def]
+def test_redshifts_from_nz(rng):  # type: ignore[no-untyped-def]
```
Comment (paddyroddy) [inline tests/test_galaxies.py:24]: To save me effort
```suggestion
def test_redshifts_from_nz(rng: np.random.Generator) -> None:
```

File: tests/test_galaxies.py [hunk 2]
```diff
@@ -128,7 +135,7 @@ def test_galaxy_shear(rng):  # type: ignore[no-untyped-def]
     assert np.shape(shear) == (512,)
 
 
-def test_gaussian_phz():  # type: ignore[no-untyped-def]
+def test_gaussian_phz(rng):  # type: ignore[no-untyped-def]
```
Comment (paddyroddy) [inline tests/test_galaxies.py:138]:
```suggestion
def test_gaussian_phz(rng: np.random.Generator) -> None:
```

File: tests/test_shapes.py [hunk 3]
```diff
@@ -46,12 +46,17 @@ def test_triaxial_axis_ratio(rng):  # type: ignore[no-untyped-def]
     assert np.all((qmax >= q) & (q >= qmin))
 
 
-def test_ellipticity_ryden04():  # type: ignore[no-untyped-def]
+def test_ellipticity_ryden04(rng):  # type: ignore[no-untyped-def]
```
Comment (paddyroddy) [inline tests/test_shapes.py:49]:
```suggestion
def test_ellipticity_ryden04(rng: np.random.Generator) -> None:
```

File: tests/test_shapes.py [hunk 4]
```diff
@@ -84,13 +89,19 @@ def test_ellipticity_ryden04():  # type: ignore[no-untyped-def]
     assert np.all((e.real >= -1.0) & (e.real <= 1.0))
 
 
-def test_ellipticity_gaussian():  # type: ignore[no-untyped-def]
+def test_ellipticity_gaussian(rng):  # type: ignore[no-untyped-def]
```
Comment (paddyroddy) [inline tests/test_shapes.py:92]:
```suggestion
def test_ellipticity_gaussian(rng: np.random.Generator) -> None:
```

File: tests/test_shapes.py [hunk 5]
```diff
@@ -108,13 +119,19 @@ def test_ellipticity_gaussian():  # type: ignore[no-untyped-def]
     np.testing.assert_allclose(np.std(eps.imag[n:]), 0.256, atol=1e-3, rtol=0)
 
 
-def test_ellipticity_intnorm():  # type: ignore[no-untyped-def]
+def test_ellipticity_intnorm(rng):  # type: ignore[no-untyped-def]
```
Comment (paddyroddy) [inline tests/test_shapes.py:122]:
```suggestion
def test_ellipticity_intnorm(rng: np.random.Generator) -> None:
```

**Review by paddyroddy — APPROVED** [review]
You love to see it

**Pull request merged**